app/src/main/java/com/keyneez/presentation/main/detail/DetailActivity.kt

```diff
@@ -0,0 +1,29 @@
+package com.keyneez.presentation.main.detail
+
+import android.os.Bundle
+import com.keyneez.util.binding.BindingActivity
+import com.keyneez.util.extension.setOnSingleClickListener
+import com.lab.keyneez.R
+import com.lab.keyneez.databinding.ActivityHomeDetailBinding
+
+class DetailActivity :
+    BindingActivity<ActivityHomeDetailBinding>(R.layout.activity_home_detail) {
+    override fun onCreate(savedInstanceState: Bundle?) {
+        super.onCreate(savedInstanceState)
+    }
+
+    // 뒤로가기 버튼
+    private fun initBackClickListener() {
+        binding.btnDetailBack.setOnSingleClickListener { }
+    }
+
+    // 공유 버튼
+    private fun initShareClickListener() {
+        binding.btnDetailShare.setOnSingleClickListener { }
+    }
+
+    // 좋아요 버튼
+    private fun initHeartClickListener() {
+        binding.btnDetailHeart.setOnSingleClickListener { }
+    }
```

Comment:
```suggestion
    /** 뒤로가기 버튼 */
    private fun initBackClickListener() {
        binding.btnDetailBack.setOnSingleClickListener { }
    }

    /** 공유 버튼 */
    private fun initShareClickListener() {
        binding.btnDetailShare.setOnSingleClickListener { }
    }

    /** 좋아요 버튼 */
    private fun initHeartClickListener() {
        binding.btnDetailHeart.setOnSingleClickListener { }
    }
```

/** 주석 */ 를 활용하여 함수에 대한 주석을 남기면 다른 파일에서 함수 사용 시에 미리보기에 주석에 써놓은 설명이 보인답니다!
그런데 지금 만드신 함수들은 네이밍만으로 어떤 함수인지 파악이 잘 되기 때문에 주석이 불필요해보이긴 하네요